### DISCUSSION TITLE
Instaling geode problem

### DISCUSSION THREAD

**Discussion by susigreg** [geode-sdk/sdk#904] [Help]
Instaling geode makes it so i cant open gd

**Comment by RoootTheFox** [comment]
... can you give any details?
- what device are you using?
- what operating system are you using?
- what version of GD are you on?
- what version of Geode are you trying to install?
- do you have any crashlogs?

**Reply by susigreg** [reply]
Using pc, windows 10. Version is 2.206, im trying to instal the latest versions in actions. For crash logs the last ones were before the upd.

**Reply by YellowCat98** [reply]
dont use the ones from actions, get https://github.com/geode-sdk/geode/releases/tag/v3.0.0-beta.1 its more stable than the ones from gh actions

**Reply by susigreg** [reply]
YellowCat98 it still wont open gd

**Reply by RoootTheFox** [reply]
verify file integrity in steam

**Reply by susigreg** [reply]
I think i found the problem, its error 126. Prob cus of the unknown publisher

**Reply by YellowCat98** [reply]
does it work WITHOUT geode

**Reply by susigreg** [reply]
yes

**Reply by susigreg** [reply]
i even reinstaled gd multiple times, and geode still wont work
